httpserver/http_connection.hpp

```diff
@@ -214,21 +251,13 @@ class http_connection : public std::enable_shared_from_this<http_connection> {
 
         const auto target = request_.target();
         switch (request_.method()) {
-        case http::verb::get:
-            if (target != "/retrieve") {
-                response_.result(http::status::not_found);
-                break;
-            }
-
-            process_retrieve();
-            break;
         case http::verb::post:
-            if (target != "/store") {
+            if (target != "/v1/storage_rpc") {
```

Comment:
Maybe this should be
```
if (target == "/v1/storage_rpc") {
  process_v1();
  break;
}
response_.result(http::status::not_found);
break;
```
Easier to add others and more straight forward logically imo

Reply:
Good point! Fixed in last commit.